Eatery/Controllers/Eateries/EateriesSharedViewController.swift

```diff
@@ -16,25 +16,23 @@ class EateriesSharedViewController: UIViewController {
     // Scroll
 
     private var lastContentOffset: CGFloat = 0
-    
     private var showPillOnScrollStopTimer: Timer?
 
     // View Controllers
 
-    private(set) lazy var campusEateriesViewController = CampusEateriesViewController()
-    private(set) lazy var collegetownEateriesViewController = CollegetownEateriesViewController()
-
-    private(set) lazy var pillViewController = PillViewController(
-        leftViewController: campusEateriesViewController,
-        rightViewController: collegetownEateriesViewController
+    private lazy var campusEateries = CampusEateriesViewController()
+    private lazy var campusNavigation = EateryNavigationController(rootViewController: campusEateries)
+    private lazy var collegetownEateries = CollegetownEateriesViewController()
```

Comment:
can we add `VC` or something like that to the end of the name just to make it clear